sdk/keyvault/Azure.Security.KeyVault.Secrets/CHANGELOG.md

```diff
@@ -1,10 +1,6 @@
 # Release History
 
-## 4.6.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 4.6.0-beta.1 (2023-11-09)
 
 - Renamed tags reported on `SecretClient` activities to follow OpenTelemetry attribute naming conventions:
```

Comment:
Not sure if I'm reading the diff wrong, but this section appears to be missing a subheading. Is that intentional?